### PR TITLE
Snapshot flipping bug fix

### PR DESCRIPTION
**Pull request by Garyfallidis**
This PR fixes the issue reported in https://github.com/fury-gl/fury/issues/467

Basically, we were not flipping the images correctly in ``save_image`` and ``load_image``. ``save_image`` was called by ``snapshot``.

### CODE REVIEW

File: fury/io.py [hunk 1]
```diff
@@ -77,6 +78,7 @@ def load_image(filename, as_vtktype=False, use_pillow=True):
             vtk_image.SetSpacing(1.0, 1.0, 1.0)
             vtk_image.SetOrigin(0.0, 0.0, 0.0)
             arr_tmp = np.flipud(image)
```
Comment (skoudoro) [inline fury/io.py:80]: Can you remove this line?  It is not used since you do `arr_tmp = image` in the line below (81)

**Review by skoudoro — COMMENTED** [review]
LGTM! I will merge it as soon as you fix my comment below. Thanks!

File: fury/io.py (hunk 1 above)
Comment (Garyfallidis) [reply fury/io.py]: Done!

**Comment by skoudoro** [comment]
Failing CI's not related, thank @Garyfallidis. merging

**Pull request merged**